docs/compression_algorithms/post_training/Quantization.md

```diff
@@ -0,0 +1,95 @@
+## Post-Training Quantization
+
+Post-Training Quantization is the quantization algorithm which doesn't demand retraining of a quantized model.
+It utilizes a small subset of the initial dataset to calibrate quantization constants.
+
+NNCF provides an advanced Post-Training Quantization algorithm, which consists of following techniques:
```

Comment:
```suggestion
NNCF provides an advanced Post-Training Quantization algorithm, that consists of following techniques:
```

Reply:
I prefer to say that it is `consists of`, Because PTQ is just a container for the algorithms.

Reply:
In this case, I think "which" is correct as its an additional information: https://www.dictionary.com/e/that-vs-which/